index.rst

```diff
@@ -6,24 +6,474 @@
+The contents of cells two through five are used to generate a table of notebook metadata in the README.md file for the repository.
+
+The sixth and seventh markdown cells should contain the credits and acknowledgements, and information about where users go to get support.  
+
+
+First section
+^^^^^^^^^^^^^
+
+The structure of the first section is mandatory.
+
+Provide a brief narrative about this notebook, e.g., "This notebook will teach the user...".
+Cite or link to any external information or documentation, and cross-reference to other notebooks.
+
+The first sub-section should always be "1.1 Package Imports".
+It should have a markdown cell that provides explanations and/or links to external package documentation, as appropriate.
+All package imports must be done in the first code cell.
+
+The second sub-section should always be "1.2 Define Functions and Parameters", if applicable.
+Globally defined utility functions, plotting defaults, or constants should be here.
+It is OK to rename the subsection to be more specific to the notebook, and/or to use sub-sub-sections like
+"1.2.1 Define global cosmological parameter values" or "1.2.2 Define a function to make an image cutout".
+It is OK to remove sub-section 1.2 if it is not used.
+
+
+Section structure
+^^^^^^^^^^^^^^^^^
+
+For all sections after the first, use numbers for sections, sub-sections, and sub-sub-sections to enable referencing in support requests,
+e.g., "I'm having trouble with the second code cell in Section 2.3."
+
+Use section titles that actively describe what is being done, e.g., "2.2 Create a color-magnitude diagram" instead of "2.2 Plot", so that the auto-generated table of contents is easy to navigate.
+
+It is very common, but not mandatory, to end all notebook tutorials with a section called "Exercises for the learner" with suggestions of
+how the user can make changes to the tutorial test options, or take the next step forward on their own.
+
+
+
+Tables and plots
+----------------
+
+Table data format
+^^^^^^^^^^^^^^^^^
+
+Results from a TAP service search are best displayed as an astropy table using ``.to_table()``,
+or as a pandas dataframe using ``.to_table().to_pandas()``.
+However, do not use the ``.to_table().show_in_notebook()`` method.
+This can cause issues in the RSP JupyterLab environment that make the notebook hang indefinitely.
+
+Plot color palette
+^^^^^^^^^^^^^^^^^^
+
+To be colorblind-friendly, plots should use the matplotlib color tables viridis or
+`cividis <https://matplotlib.org/stable/users/prev_whats_new/whats_new_2.2.html#cividis-colormap>`_ (or a greyscale),
+or the new `tableau-colorblind10 <https://viscid-hub.github.io/Viscid-docs/docs/dev/styles/tableau-colorblind10.html>`_. 
+
+:: 
+
+  import matplotlib.pyplot as plt
+  plt.style.use('tableau-colorblind10')
+
+
+For the LSST filter set ugrizy, adopt the same colors as DES, which were chosen to be colorblind-friendly: 
```

Comment:
Spell out DES the first time it is used. Should "ugrizy" be in a different font?

Reply:
done